esmvaltool/cmorizers/data/datasets.yml

```diff
@@ -1,6 +1,14 @@
 # Dataset information
 ---
 datasets:
+  AGCD:
+    tier: 3
+    source: "https://dx.doi.org/10.25914/hjqj-0x55"
+    last_access: 2023-11-21
+    info: |
+      Data from National Computing Infrastructure(NCI) project requiring an NCI account and access to Gadi, processing is done on Gadi.
```

Comment:
is this something only a very restricted group of people could access? ie what is Gadi and how does one get access to it (whatever it is)?

Reply:
I have updated the info for the AGDC dataset. @flicj191 your doi linked to v1. I think this should be v2 right?

@valeriupredoi Gadi is the NCI supercomputer.

Reply:
The NCI catalog is public but users need to request access if they want to use the datasets.

Reply:
gotcha, it'd be nice to explain what Gadi is in the docstring (just that, what Romain mentioned) pls